lib/charms/mongodb_libs/v0/mongodb_provider.py

```diff
@@ -159,7 +322,7 @@ def _diff(self, event: RelationChangedEvent) -> Diff:
             key for key in old_data.keys() & new_data.keys() if old_data[key] != new_data[key]
         }
 
-        # TODO: update when evaluatation of the possibility of losing the diff is completed 
+        # TODO: update when evaluatation of the possibility of losing the diff is completed
```

Comment:
evaluatation -> evaluation

Reply:
seems like I was ignoring all lib code in tox, (so I'll update tox to listen to mongodb lib code)